docs/setup/windows-setup.md

```diff
@@ -1,30 +1,37 @@
-## NB! This manual is for running backend without docker, if you don't plan to debug back-end it may be easier to follow docker route: [Understand rtc build and deployments guide](../../deployment/understand-rtc-build-and-deployments.md)
+## NB! This manual is for running backend without docker, if you don't plan to debug back-end it may be easier to follow docker route: [Understand rtc build and deployments guide](../deployment/readme.md)
 
 #### Client
+
 For the web client follow these [instructions](../client.md)
 
 ## Running with Visual Studio 2019 on Windows
 
 Install Visual Studio 2019 Pro or Community. Upon running the installation, be sure to install the following optional components:
-  - .NET Framework 4.6.2
-  - .NET Core 2.1 development tools
+
+- .NET Framework 4.6.2
```

Comment:
Do we really need .NET Framework? It is not needed for the other environments...